dragonflybsd/DragonFlyBSDProcessTable.c

```diff
@@ -249,44 +249,56 @@ void ProcessTable_goThroughEntries(ProcessTable* super) {
       // would be nice if we could store multiple states in proc->state (as enum) and have writeField render them
       /* Taken from: https://github.com/DragonFlyBSD/DragonFlyBSD/blob/c163a4d7ee9c6857ee4e04a3a2cbb50c3de29da1/sys/sys/proc_common.h */
       switch (kproc->kp_stat) {
```

Comment:
Likewise. The indent style of `case` statements inside `switch` are not consistent even after this PR.

Reply:
They should be now … ;-) Otherwise drop me a note and I'll re-check.